### ISSUE TITLE
Telescopetest-io: add accessability

### ISSUE THREAD

**Issue by Judyzc** [cloudflare/telescope#185]
Telescopetest-io should be screen reader and keyboard accessible, and all text/visualizations should have good contrast. Maybe we can use tools like [WAVE](https://wave.webaim.org/) to help out.
